node/src/components/storage/tests.rs

```diff
@@ -38,6 +38,23 @@ use crate::{
 
 type BlockGenerators = Vec<fn(&mut TestRng) -> (Block, EraId)>;
 
+impl Storage {
+    fn disjoint_sequences(&self) -> &Vec<Sequence> {
+        self.disjoint_block_height_sequences.sequences()
+    }
+
+    fn add_missing_block_body(&mut self, block_header: &BlockHeader) {
+        match self.missing_block_bodies.entry(*block_header.body_hash()) {
+            Entry::Occupied(mut entry) => {
+                entry.get_mut().push(block_header.height());
+            }
+            Entry::Vacant(entry) => {
+                entry.insert(vec![block_header.height()]);
+            }
+        }
+    }
```

Comment:
Same suggestion with `or_default()`.

Also, maybe we should just make it a method in production code and call it in other places where this is duplicated?